schema/config-schema.json

```diff
@@ -118,6 +118,10 @@
                 "user": {
                     "id": "https://opencontainers.org/schema/bundle/process/user",
                     "type": "object",
+                    "required": [
+                        "uid",
+                        "gid"
```

Comment:
We can't add this to the JSON Schema, because they're only required on Linux.  [Windows entries will not have them][1].

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc4/config.md#windows-user